samples/min-logging/README.md

```diff
@@ -4,20 +4,17 @@ This sample demonstrates how you can customize a logging deployment
 to minimize resource usage. The sample deployment configures single instances of each logging 
 component. This configuration could save CPU and memory resources and could be useful in development and test environments.
 
-## Installation
+## Using This Sample
 
-Follow these steps:
+You customize your logging deployment by specifying values in `user.env` and `*.yaml` files. These files are stored in a local directory outside of your repository that is identified by the `USER_DIR` environment variable. The configuration files in this sample provide a starting point for the configuration files for a deployment that supports multi-role Elasticsearch nodes. See the 
+[main README](../../README.md#customization) to for information about the customization process.
```

Comment:
Second sentence looks like some cut-and-paste text from the multi-role Elasticsearch sample has not been tweaked for to discuss this particular sample.
Maybe we should consolidate the "boilerplate" discussion of customization into one paragraph and then add the sample-specific text (generally a single sentence explaining what this particular sample is showing) in a 2nd paragraph (or as the last sentence in the boilerplate).  Right now, I think it is too easy to completely miss the "what is this sample" info because it is embedded within a paragraph that "looks familiar".    I think all this would mean is moving the last sentence (pointer to main README) to the 2nd sentence in the paragraph.

Reply:
Done. I moved the sample-specific text to a separate paragraph below the boilerplate text (done in all samples)